bip-0325.mediawiki

```diff
@@ -60,8 +60,10 @@ The "to_sign" transaction is:
     vin[0].prevout.hash = to_spend.txid
     vin[0].prevout.n = 0
     vin[0].nSequence = 0
+    vin[0].sigScript = [ signet_solution first data push, if any ]
+    vin[0].scriptWitness = [ signet_solution second data push, if any ]
```

Comment:
```suggestion
    vin[0].scriptWitness = [ signet_solution scriptWitness (y bytes), if any ]
```

Maybe refer to it with the length to avoid the "data push" confusion?

Reply:
Good idea